lib/fake_idp/application.rb

```diff
@@ -84,39 +82,5 @@ def attributes_xml(attributes)
         %[<saml:Attribute Name="#{name}">#{attribute_value}</saml:Attribute>]
       end
     end
-
-    # This method overrides its definition in the ruby-saml-idp gem
-    def encode_SAMLResponse(nameID, opts = {})
```

Comment:
Why isn't this needed anymore?

Reply:
I removed this and the line requiring the encryptor because I'm splitting that work into other PRs. So this PR is primarily introducing the Encryptor class but not using it yet and the cleanup you did some time ago to support `encryption_enabled` options. The `encode_SAMLResponse` method is going to be handled by https://github.com/healthify/fake_idp/pull/20 which introduces the SAML response generator after which will be extended to support encrypted assertions.